### PR TITLE
Revisión Softwarereview reviewer.es.rmd es

### PR DESCRIPTION
**Pull request by paocorrales**
@yabellini this one is ready!

### CODE REVIEW

File: softwarereview_reviewer.es.Rmd [hunk 1]
```diff
@@ -1,86 +1,87 @@
-# Guía para revisores {#reviewerguide}
+# Guía para quienes hace una revisión {#reviewerguide}
```
Comment (yabellini) [inline softwarereview_reviewer.es.Rmd:1]:
```suggestion
# Guía para quienes hacen una revisión {#reviewerguide}
```

File: softwarereview_reviewer.es.Rmd [hunk 2]
```diff
@@ -1,86 +1,87 @@
...
 
-Por favor, rellena nuestro [formulario de voluntariado](https://ropensci.org/software-reviewer).
+Por favor, completa nuestro [formulario de voluntariado](https://ropensci.org/software-reviewer).
 
-Si ves una presentación actual que es particularmente relevante para tus intereses, por favor envía un correo electrónico a `info@ropensci.org`incluyendo el nombre del paquete, la URL del número de presentación y el nombre del editor asignado.
-Sin embargo, ten en cuenta que las invitaciones a los revisores se mantienen a discreción del editor, y es muy posible que éste ya haya enviado un correo electrónico a la gente.
-Por favor, no te ofrezcas como voluntario para todos los temas, y no te ofrezcas a través de la interfaz de GitHub.
+Si ves una presentación vigente que es particularmente relevante para tus intereses, por favor envía un correo electrónico a `info@ropensci.org`, incluyendo el nombre del paquete, la URL del número de presentación y el nombre de la persona responsable de la edición.
+Sin embargo, ten en cuenta que las invitaciones a revisión quedan a discreción de la persona a cargo de la edición, y es muy posible que ya haya realizado invitaciones.
```
Comment (yabellini) [inline softwarereview_reviewer.es.Rmd:26]:
```suggestion
Sin embargo, ten en cuenta que las invitaciones a revisar quedan a discreción de la persona a cargo de la edición, y es muy posible que ya haya realizado invitaciones. 
```

File: softwarereview_reviewer.es.Rmd [hunk 3]
```diff
@@ -1,86 +1,87 @@
...
-- ¿Se nombraron las funciones y los argumentos para que funcionen juntos y formen una API de programación común y lógica que sea fácil de leer y autocompletar?
-- Si tienes tus propios datos/problema relevantes, trabaja con el paquete. Puede que encuentres asperezas y casos de uso en los que el autor no haya pensado.
+- ¿Puede mejorarse el estilo del código?
+- ¿Puede reducirse el código duplicado, de haberlo?
+- ¿Podría mejorarse la interfaz de usuario?
+- ¿Podría aumentarse el rendimiento?
+- ¿La documentación (instrucciones de instalación, viñetas, ejemplos, demos) es clara y suficiente? ¿Utiliza el principio de *múltiples puntos de entrada*? Es decir, tiene en cuenta que cualquier parte de la documentación puede ser la primera exposición de una persona con el paquete y/o la herramienta o datos que utiliza?
+- ¿Los nombres de funciones y argumentos forman una API de programación común y lógica que sea fácil de leer y que se beneficie de herramientas de automcompletado?
+- Si tienes tus propios datos relevante o un problema relacionado, trabaja con el paquete. Puede que encuentres asperezas y/o casos de uso no contemplados.
```
Comment (yabellini) [inline softwarereview_reviewer.es.Rmd:47]:
```suggestion
- Si tienes datos propios relevantes o un problema relacionado, trabaja con el paquete. Puede que encuentres asperezas y/o casos de uso no contemplados.
```

Comment (yabellini) [reply softwarereview_reviewer.es.Rmd:47]: asperezas?

File: softwarereview_reviewer.es.Rmd [hunk 4]
```diff
@@ -1,86 +1,87 @@
...
+Todos los envíos generan un informe detallado sobre la estructura y funcionalidad del paquete, generado por [nuestro paquete `pkgcheck`](https://docs.ropensci.org/pkgcheck/). Si el paquete ha cambiado sustancialmente desde las últimas revisiones, puedes solicitar una nueva revisión con el comando `@ropensci-review-bot check package`. Ten en cuenta que, al instalar el paquete para revisarlo, debes utilizar el argumento `dependencies = TRUE` de `remotes::install()` para asegurarte de que tienes todas las dependencias necesarias.
 
 ### Directrices generales
 
-Para revisar un paquete, empieza por copiar nuestra [plantilla de revisión](#reviewtemplate) (o nuestra [plantilla de revisión en español](#reviewtemplatees)) y utilízala como lista de control de alto nivel.  Además de marcar los criterios mínimos, te pedimos que hagas comentarios generales sobre lo siguiente
+Para revisar un paquete, empieza por copiar nuestra [plantilla de revisión](#reviewtemplate) (o nuestra [plantilla de revisión en español](#reviewtemplatees)) y utilízala como checklist general. Además de marcar los criterios mínimos, te pedimos que hagas comentarios generales sobre lo siguiente:
```
Comment (yabellini) [inline softwarereview_reviewer.es.Rmd:37]:
```suggestion
Para revisar un paquete, empieza por copiar nuestra [plantilla de revisión](#reviewtemplate) (o nuestra [plantilla de revisión en español](#reviewtemplatees)) y utilízala como lista de verificación general. Además de marcar los criterios mínimos, te pedimos que hagas comentarios generales sobre lo siguiente:
```

Comment (yabellini) [reply softwarereview_reviewer.es.Rmd:37]: En T3 la traducción de checklist fue listas de verificación

File: softwarereview_reviewer.es.Rmd [hunk 5]
```diff
@@ -1,86 +1,87 @@
...
+- Ejecutar `devtools::check()` y `devtools::test()` en el paquete para encontrar cualquier error que haya 
+pasado desapercibido en la computadora de quien desarrolló el paquete.
+- Revisar si los tests salteados son justificados (por ejemplo usar `skip_on_cran()` en tests que usan una API versus saltearse todos los tests en un determinado sistema operativo).
+- Si el paquete no se envía a través de la branch principal o por defecto, recuerda cambiar a la branch enviada para revisar antes de comenzar tu revisión. En este caso, tendrás que usar la búsqueda local para revisar el código, ya que la búsqueda en GitHub está limitada a la branch por defecto. Además, la documentación alojada en un sitio web de `pkgdown` no va a estar necesariamente actualizada, por lo que recomendamos revisar la documentación del paquete localmente ejecutando `pkgdown::build_site()`.
```
Comment (yabellini) [inline softwarereview_reviewer.es.Rmd:58]:
```suggestion
- Si el paquete no se envía a través de la _branch_ principal o por defecto, recuerda cambiar a la _branch_ enviada para revisar antes de comenzar tu revisión. En este caso, tendrás que usar la búsqueda local para revisar el código, ya que la búsqueda en GitHub está limitada a la _branch_ por defecto. Además, la documentación alojada en un sitio web de `pkgdown` no va a estar necesariamente actualizada, por lo que recomendamos revisar la documentación del paquete localmente ejecutando `pkgdown::build_site()`.
```

File: softwarereview_reviewer.es.Rmd [hunk 6]
```diff
@@ -1,86 +1,87 @@
...
 
-### Experiencia de revisores anteriores
+### Experiencia de revisiones anteriores
 
-A los revisores noveles les puede resultar útil leer (sobre) algunas revisiones anteriores. En general, puedes encontrar hilos de presentación de paquetes incorporados [aquí](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aclosed+label%3A6%2Fapproved). Aquí tienes algunos ejemplos escogidos de reseñas (ten en cuenta que tus reseñas no tienen que ser tan largas como estos ejemplos):
+Si es la primera vez que revisas un paquete, te puede ser útil leer algunas revisiones anteriores. En general, puedes encontrar *issues* de presentación de paquetes aceptados [aquí](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aclosed+label%3A6%2Fapproved). Estos son algunas revisiones seleccionadas en inglés (ten en cuenta que tus reseñas no tienen que ser tan largas como estos ejemplos):
```
Comment (yabellini) [inline softwarereview_reviewer.es.Rmd:68]:
```suggestion
Si es la primera vez que revisas un paquete, te puede ser útil leer algunas revisiones anteriores. En general, puedes encontrar *issues* de presentación de paquetes aceptados [aquí](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aclosed+label%3A6%2Fapproved). Estas son algunas revisiones seleccionadas en inglés (ten en cuenta que tus reseñas no tienen que ser tan largas como estos ejemplos):
```

File: softwarereview_reviewer.es.Rmd [hunk 7]
```diff
@@ -1,86 +1,87 @@
...
+Puedes leer las entradas del blog sobre experiencias al revisar paquetes [a través de este enlace](https://ropensci.org/tags/reviewer/). En particular [esta publicación de Mara Averick](https://ropensci.org/blog/2017/08/22/first-package-review/) (en inglés) habla como puedes proporcionar una devolución útil aún careciendo de experticia en el tema o en la implementación del paquete adoptando el rol de "usuario/a ingenuo/a" y preguntándote *"¿Qué creo que hace esta cosa? ¿lo hace? ¿qué cosas me causan rechazo?"* En [otra entrada del blog](https://ropensci.org/blog/2017/09/08/first-review-experiences/) Verena Haunschmid explica cómo alternó entre usar el paquete y revisar el código.
```
Comment (yabellini) [inline softwarereview_reviewer.es.Rmd:78]:
```suggestion
Puedes leer los artículos del blog sobre experiencias al revisar paquetes [a través de este enlace](https://ropensci.org/tags/reviewer/). En particular [esta publicación de Mara Averick](https://ropensci.org/blog/2017/08/22/first-package-review/) (en inglés) habla como puedes proporcionar una devolución útil aún careciendo de experticia en el tema o en la implementación del paquete adoptando el rol de "usuario/a ingenuo/a" y preguntándote *"¿Qué creo que hace esta cosa? ¿lo hace? ¿qué cosas me causan rechazo?"* En [otra entrada del blog](https://ropensci.org/blog/2017/09/08/first-review-experiences/) Verena Haunschmid explica cómo alternó entre usar el paquete y revisar el código.
```

File: softwarereview_reviewer.es.Rmd [hunk 8]
```diff
@@ -89,27 +90,28 @@ pkgreview_create(pkg_repo = "embruna/refnet",
...
-y
+Esto
 
-- el repositorio de GitHub del paquete `refnet` paquete se clonará.
-- se creará un proyecto de revisión, que contendrá un cuaderno para que lo rellenes, y la plantilla de revisión.
-- ten en cuenta que si el paquete no se envía a través de la rama principal/por defecto, tienes que cambiar a la rama enviada antes de comenzar tu revisión.
+- clonará el repositorio de GitHub del paquete `refnet`,
+- creará un proyecto de revisión, que contendrá una notebook para que rellenes, y la plantilla de revisión.
+
+Ten en cuenta que si el paquete no se envía a través de la branch principal o por defecto, tienes que cambiar a la branch usada para enviar antes de comenzar tu revisión.
```
Comment (yabellini) [inline softwarereview_reviewer.es.Rmd:98]:
```suggestion
Ten en cuenta que si el paquete no se envía a través de la _branch_ principal o por defecto, tienes que cambiar a la _branch_ usada para enviar antes de comenzar tu revisión.
```

**Review by yabellini — APPROVED** [review]
Minor editions and one question. Thanks!

**Comment by paocorrales** [comment]
Yani I don't see the question... :)

**Comment by yabellini** [comment]
It was about the use of the word "asperezas", but I couldn't think of a better option.

**Pull request merged**